src/main/java/core/basesyntax/processing/CsvStrategyImpl.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.processing;
+
+import core.basesyntax.model.Apple;
+import core.basesyntax.model.Banana;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Orange;
+import core.basesyntax.service.StorageService;
+import core.basesyntax.service.StorageServiceImpl;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvStrategyImpl implements CsvStrategy {
+    private int bananaCount;
+    private int appleCount;
+    private int orangeCount;
+
+    CsvStrategyImpl() {
+        bananaCount = 0;
+        appleCount = 0;
+        orangeCount = 0;
+    }
+
+    @Override
+    public boolean countFruit(Fruit fruit) {
+        if (fruit.getClass().equals(Banana.class)) {
+            bananaCount++;
+            return true;
+        } else if (fruit.getClass().equals(Apple.class)) {
+            appleCount++;
+            return true;
+        } else if (fruit.getClass().equals(Orange.class)) {
+            orangeCount++;
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean chooseOperation(String[] operation) {
```

Comment:
Bogdan said that its not the best solution to use switch here. May be it's better to create different classes for each operation? and Map<Char, Operation> as choosing mechanism